lib/configsetup.inc.php

```diff
@@ -152,16 +158,19 @@
 		'jpeg_quality_image' => [
 			'type' => 'input',
 			'name' => 'jpeg_quality_image',
+			'placeholder' => '80',
```

Comment:
We could also use `$defaultConfig['jpeg_quality_image']` at this point, so there is no inconsistency if someone changes the default value.

Reply:
Then we should do that on all in future. (E.g. also on `cheese_time` etc).
For now I'd merge it as it is if there's nothing against it.

You're ok with that?